src/utils/theme.js

```diff
@@ -13,6 +14,9 @@ const theme = createMuiTheme({
     text: {
       primary: 'rgba(0, 0, 0, 0.80)',
     },
+    backgroundContrastText: {
```

Comment:
Let's add a comment here calling out that this is a custom property (not part of MUI), in case this causes problems/confusion in the future.